KsApi/mutations/templates/ClearUserUnseenActivityMutationTemplate.swift

```diff
@@ -0,0 +1,33 @@
+import Apollo
+@testable import KsApi
+
+public enum ClearUserUnseenActivityMutationTemplate {
```

Comment:
Curious why you modeled this as an enum that then accesses a `data` property? 

Our other templates in the project are basically just constants that are set to objects that are filled with mock data.

Reply:
Yea I guess I was thinking about writing two types of returnable data in the `Templates` related to `GraphAPI`. For one thing we just case two cases `valid` and `errored`. We can also pass in parameters to customize our data like in `CreateBackingMutationTemplate` we can pass in a `CheckoutState` or `sca` bool into the enum type and get back valid data customize those paramters. It just simplifies the call a little bit so we don't need duplicate sets of huge dictionaries that have minor differences. Let me know if you prefer the other approach.